src/Sarif/Writers/ResultLogJsonWriter.cs

```diff
@@ -130,7 +127,7 @@ public void Initialize(Run run)
                 _serializer.Serialize(_jsonWriter, run.DefaultFileEncoding);
             }
 
-            if (run.RichMessageMimeType != null)
+            if (run.RichMessageMimeType != null && run.RichMessageMimeType != "text/markdown;variant=GFM")
```

Comment:
>"text/markdown;variant=GFM [](start = 78, length = 26)

here's a place where it would be nice to reference a jschema-generated constant value.

Reply:
Like, if a property `MyProp` of type `T` has default `X`, then generate this in the class:
```
public const T DefaultMyProp = X;
```
In this example, `Run.DefaultRichMessageType` of type `string`.

That is a great idea (and I see you already filed jschema#90 for it).

---
In reply to: [246175862](https://github.com/Microsoft/sarif-sdk/pull/1204#discussion_r246175862) [](ancestors = 246175862)